CONTRIBUTING.md

```diff
@@ -95,7 +95,13 @@ Key: Value
 ```
 
 When a commit has related issues or commits, explain the relation in the message
-body. You should also leave an `Issue` tag in the footer. For example:
+body. You should also leave an `Issue` tag in the footer. However, if this is the
+final commit that fixes an issue you should leave a `Closes` tag in the footer
```

Comment:
There is more than one tag which will automatically close the PR.  Maybe change the text to `Closes` or one of the tags found [here](https://help.github.com/articles/closing-issues-using-keywords/).

Reply:
Fixed.